package.json

```diff
@@ -10,7 +10,7 @@
     "metabolica-core": "github:biosustain/metabolica-ui-core",
     "metabolica-home": "github:DD-DeCaF/metabolica-ui-home#v1.0.2",
     "metabolica-login": "github:DD-DeCaF/metabolica-ui-login#v1.0.2",
-    "metabolica-map": "github:DD-DeCaF/metabolica-ui-map#v1.2.1",
+    "metabolica-map": "github:DD-DeCaF/metabolica-ui-map#devel",
```

Comment:
Please don't merge devel dependencies into master!

Reply:
This one is not perfectly figured out. On devel branch we are referencing the devel dependencies, on master the versioned ones. I'm not sure if it's possible to dispatch this in package.json